get-options.sh

```diff
@@ -27,13 +27,17 @@ show_help() {
          usage: ./docker-run.sh --platform core.x|xeon|dgpu.x --inputsrc RS_SERIAL_NUMBER|CAMERA_RTSP_URL|file:video.mp4|/dev/video0 [--classification_disabled] [ --ocr_disabled | --ocr [OCR_INTERVAL OCR_DEVICE] ] [ --barcode_disabled | --barcode [BARCODE_INTERVAL] ] [--realsense_enabled] --workload dlstreamer|opencv-ovms
 
          Note: 
-	  1. dgpu.x should be replaced with targeted GPUs such as dgpu (for all GPUs), dgpu.0, dgpu.1, etc
-	  2. core.x should be replaced with targeted GPUs such as core (for all GPUs), core.0, core.1, etc
-	  3. filesrc will utilize videos stored in the sample-media folder
-	  4. Set environment variable STREAM_DENSITY_MODE=1 for starting single container stream density testing
-	  5. Set environment variable RENDER_MODE=1 for displaying pipeline and overlay CV metadata
-	  6. Set environment variable LOW_POWER=1 for using GPU usage only based pipeline for Core platforms
-	  7. Set environment variable CPU_ONLY=1 for overriding inference to be performed on CPU only
+         1. dgpu.x should be replaced with targeted GPUs such as dgpu (for all GPUs), dgpu.0, dgpu.1, etc
+         2. core.x should be replaced with targeted GPUs such as core (for all GPUs), core.0, core.1, etc
+         3. filesrc will utilize videos stored in the sample-media folder
+         4. Set environment variable STREAM_DENSITY_MODE=1 for starting pipeline stream density testing
+         5. Set environment variable RENDER_MODE=1 for displaying pipeline and overlay CV metadata
+         6. Set environment variable LOW_POWER=1 for using GPU usage only based pipeline for Core platforms
+         7. Set environment variable CPU_ONLY=1 for overriding inference to be performed on CPU only
+         8. Set environment variable PIPELINE_PROFILE=\"grpc_go\" for choosing opencv-ovms workload's pipeline profile to run, values can be listed by \"make list-profiles\"
+         9. Set environment variable STREAM_DENSITY_FPS=15.0 for setting stream density target fps value
+         10. Set environment variable STREAM_DENSITY_INCREMENTS=1 for setting incrementing number of pipelines for running stream density
+         11. Set environment variable DEVICE=\"CPU\" for setting device to use for pipeline run, value can be \"GPU\", \"CPU\", \"MULTI:GPU,CPU\"
```

Comment:
GPU.x should be added to the list. This will work the same as the dgpu.x option above.

Reply:
as we discussed, We don't need to add GPU.x for now, but we will add auto